loadtest/store/memstore/random.go

```diff
@@ -68,12 +92,22 @@ func (s *MemStore) RandomTeamMember(teamId string) (model.TeamMember, error) {
 			break
 		}
 	}
-	key := pickRandomKeyFromMap(teamMemberMap).(string)
-	return *teamMemberMap[key], nil
+	key, err := pickRandomKeyFromMap(teamMemberMap)
+	if err != nil {
+		return model.TeamMember{}, err
+	}
+	return *teamMemberMap[key.(string)], nil
 }
 
-func pickRandomKeyFromMap(m interface{}) interface{} {
-	keys := reflect.ValueOf(m).MapKeys()
+func pickRandomKeyFromMap(m interface{}) (interface{}, error) {
+	val := reflect.ValueOf(m)
+	if val.Kind() != reflect.Map {
+		return nil, errors.New("not a map")
```

Comment:
Is there a reason for not using a named error here?

Reply:
It's an internal error. There is no way to pass a parameter which will cause `pickRandomKeyFromMap` to get something else other than a map. The error can indeed bubble up, but it's unlikely. So there isn't much use to naming it.